src/dso_api/dynamic_api/routers.py

```diff
@@ -139,7 +170,7 @@ def _initialize_viewsets(self) -> List[Type[DynamicModel]]:
         # Atomically copy the new viewset registrations
         self.registry = self.static_routes + dataset_routes + remote_routes
         self._openapi_urls = openapi_urls
-        self._dataset_ids = [ds.schema.id for ds in db_datasets + api_datasets]
+        self._dataset_ids = [ds.id for ds in db_datasets + api_datasets]
```

Comment:
uhm... dit zijn wel echt 2 verschillende dingen.. het model ID (primary key = integer), of de naam van het schema. Klopt dit wel?

Reply:
Het is een _ property die in de class zelf niet verder wordt aangeroepen. Dan moet het oke zijn toch? Het zijn nu echt de ids van de Dataset modellen ipv dataset.schema vond qua naamgeving duidelijker, raakte daardoor eerst in de war.

Reply:
hmm, wacht, je hebt gelijk. je hebt de `DynamicAPIRootView` natuurlijk vervangen, dus er gaat geen code stuk!